web/styles/popovers.css

```diff
@@ -568,37 +568,25 @@ ul {
 
     .user_profile_manage_own_edit_button,
     .user_profile_manage_others_edit_button {
```

Comment:
Since at least one of these are being renamed, a nice prep commit would use the hyphenated style we're moving towards, instead of replicating/carrying forward the underscore style.